server/chat-plugins/poll.js

```diff
@@ -306,7 +307,7 @@ const commands = {
 				if (room.poll.timeout) clearTimeout(room.poll.timeout);
 				room.poll.timeoutMins = timeout;
 				room.poll.timeout = setTimeout(() => {
-					room.poll.end();
+					if (room.poll) room.poll.end();
 					delete room.poll;
```

Comment:
`room.poll = null` probably?

Reply:
Wow, yeah, I'm surprised this passed the linter.

```suggestion
					room.poll = null;
```